### PR TITLE
45 update README

### PR DESCRIPTION
**Pull request by nmaarnio**

### CODE REVIEW

**Comment by nmaarnio** [comment]
I revised the README and tagged you three as reviewers, but all of you don't necessarily need to comment or others who find their way here can also comment!

**Review by lehtonenp — COMMENTED** [review]
The text is good and easy to read. I especially like how the notes, tips and warning are highlighted. The thing I would rephrase is this sentence 'Ideas for new fetures are welcome, even if we have a vision of the final version and its extent in our mind' to 'Ideas for new features are welcome, even if we already have a vision of the final version and its scope'. Note the misspelling of features in the original sentence.

I was also thinking whether at the beginning we should introduce the acronym EIS as follows: Exploration Information System (EIS), instead of EIS (_Exploration Information Systems_). I noticed that in the logo it is System, not Systems.

**Comment by nmaarnio** [comment]
Good suggestions and findings, thanks! I'll update the README. I agree with you that EIS could be introduced the way you suggested, but since the the first sentence it appears uses it as part of the name of the plugin (EIS QGIS Plugin), I am not sure could we write it "Exploration Information System (EIS) QGIS Plugin" as it might change its meaning then.

**Review by lehtonenp — APPROVED** [review]
LGTM!

**Comment by nmaarnio** [comment]
Merging now, for further improvements let's just make a new PR
